cells/lib/ops/mkStandardOCI.nix

```diff
@@ -68,19 +68,43 @@ in
       then debugOperable
       else operable;
 
-    setupLinks = cell.ops.mkSetup "links" [] ''
-      mkdir -p $out/bin
-      ${runtimeEntryLink}
-      ${debugEntryLink}
-      ${livenessLink}
-      ${readinessLink}
-    '';
+    setupLinks =
+      cell.ops.mkSetup "links" [
+        {
+          regex = "/bin";
+          mode = "0555";
+        }
+      ] ''
+        mkdir -p $out/bin
+        ${runtimeEntryLink}
+        ${debugEntryLink}
+        ${livenessLink}
+        ${readinessLink}
+      '';
+
+    setupBareMinimumEnv =
+      cell.ops.mkSetup "env" [
+        {
+          regex = "/tmp";
+          mode = "0777";
+        }
+      ] ''
+        mkdir -p $out/tmp
+        # Setup certificates
+        cp -r ${nixpkgs.cacert}/* $out
+      '';
+
+    setupSparseUser = cell.ops.mkUser {
+      user = "nobody";
+      group = "nogroup";
+      inherit uid gid;
+    };
   in
     cell.ops.mkOCI (
       {
         inherit name uid gid labels options perms config meta;
         entrypoint = operable';
-        setup = [setupLinks] ++ setup;
+        setup = [setupLinks setupBareMinimumEnv] ++ setup;
```

Comment:
I refrained from adding `/etc`, by design. However, it's symlinked by coincidence via `nixpkgs.cacert`.

But erring on the side of design, I consider it may be slightly below P95 already.